Frameworks/UIKit/UILabel.mm

```diff
@@ -313,7 +246,7 @@ - (void)_initUILabel {
     // TODO: Reevaluate whether or not this is the correct default mode for UILabels that are initialized via initWithFrame.
     //       Some of our test apps expect the initWithCoder path to default to UIViewContentModeScaleToFill (aka kCAGravityResize).
     [self setContentMode:UIViewContentModeRedraw];
-    [self _updateXamlElement];
+    [self _updateXamlElementWithAdjustedFont:nil];
```

Comment:
is there any way we can avoid this hit until our first layout pass?  It seems like this call is unnecessary?